content/en/security/infrastructure_vulnerability/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Infrastructure Vulnerability Management
+kind: documentation
+is_beta: true
+further_reading:
+ - link: "/security/infrastructure_vulnerability/setup"
+   tag: "Documentation"
+   text: "Setting up Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] uses the Datadog Agent to monitor your container images and identify known vulnerabilities for you to prioritize and remediate. The Agent scans running container images on hosts and creates a software bill of materials (SBOM). The SBOM is then scanned for vulnerabilities.
+
+## Review the health of your container images
+
+View a list of running container images on the [Container Images][2] page. Identify container images with vulnerabilities and view additional details about the vulnerability on the details page.
+
+{{< img src="security/infrastructure_vulnerability/container_images.png" alt="Container Images overview page" width="100%">}}
```

Comment:
Can we please use the screenshot where the view (Infrastructure --> Containers --> Container Images) is sorted (desc) by "CONTAINERS" column? This way, the hottest container images are surfaced on the top. They have the most vulnerabilities.